04_verify.sh

```diff
@@ -242,7 +242,9 @@ echo ""
 if [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
   iterate check_k8s_entity deployments "${EXPTD_V1ALPHA4_DEPLOYMENTS}"
   iterate check_k8s_rs "${EXPTD_V1ALPHA4_RS}"
-# TODO: add else condition when v1alpha5 is available.
+else
+  iterate check_k8s_entity deployments "${EXPTD_V1ALPHA5_DEPLOYMENTS}"
+  iterate check_k8s_rs "${EXPTD_V1ALPHA5_RS}"
```

Comment:
It is also necessary to add` EXPTD_V1ALPHA5_DEPLOYMENTS` and `EXPTD_V1ALPHA5_RS` at line 204. You can follow the example for `EXPTD_V1ALPHA4_`